packages/wallets/src/features/cfd/flows/CTrader/AvailableCTraderAccountsList/AvailableCTraderAccountsList.tsx

```diff
@@ -32,6 +32,7 @@ const AvailableCTraderAccountsList: React.FC = () => {
         if (status === 'success') {
             show(
                 <CTraderSuccessModal
+                    displayBalance={activeWallet?.display_balance || '10,000.00 USD'}
```

Comment:
Hi @lubega-deriv,
Instead of a hardcoded value of 10k, can we have empty string as fallback?
So that in case it breaks in future, we will get a clear indication in UI that this part is not displaying properly.

Reply:
Updated